tests/vmi_cloudinit_test.go

```diff
@@ -191,7 +191,9 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 						fedoraPassword,
 						sshAuthorizedKey,
 					)
-					vmi := tests.NewRandomVMIWithEphemeralDiskAndConfigDriveUserdataHighMemory(cd.ContainerDiskFor(cd.ContainerDiskFedoraTestTooling), userData)
+					vmi := libvmi.NewFedora(
+						libvmi.WithCloudInitConfigDriveUserData(userData),
+					)
 
 					LaunchVMI(vmi)
```

Comment:
```suggestion
					vmi = LaunchVMI(vmi)
```

to avoid `Panic: an empty namespace may not be set when a resource name is provided` as `LaunchVMI()` returns a running VMI, with namespace set.